### PR TITLE
Really fix link to Uses page history on GitHub

### PR DESCRIPTION
**Pull request by Aprillion**
Unfortunately, `master` is not redirected to `main` for this link ¯\\\_(ツ)_/¯

### CODE REVIEW

File: src/pages/uses/index.mdx [hunk 1]
```diff
@@ -274,4 +274,4 @@ https://www.youtube.com/watch?v=N3wY9kNnmQA
 ## Old setup
 
 To learn what this page looked like in the past, checkout
-[the file history on GitHub](https://github.com/kentcdodds/kentcdodds.com/commits/master/src/pages/uses.mdx)
+[the file history on GitHub](https://github.com/kentcdodds/kentcdodds.com/commits/main/src/pages/uses.mdx)
```
Comment (Aprillion) [inline src/pages/uses/index.mdx]: hmm, why is there no `uses/index.mdx` in the end after previous PR?

**Review by kentcdodds — APPROVED** [review]
Thanks!

**Comment by kentcdodds** [comment]
Whoops, merge conflict 😅

**Review by kentcdodds — APPROVED** [review]
Sweet, thanks

**Comment by Aprillion** [comment]
> Whoops, merge conflict 😅

yeah, that's what I got for trying to fix it faster by using GitHub web GUI 😆

**Pull request merged**